x11-packages/mlt/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://www.mltframework.org/
+TERMUX_PKG_DESCRIPTION="Multimedia Framework. Author, manage, and run multitrack audio/video compositions."
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_VERSION=7.24.0
```

Comment:
Actually I just noticed, the build scripts are out of date now.
Would you mind bumping `mlt` to 7.28.0